bin/jest-it-up

```diff
@@ -7,6 +7,12 @@ const jestItUp = require('../lib')
 program
   .option('-c, --config <path>', 'path to a Jest config file', 'jest.config.js')
   .option('-m, --margin <margin>', 'minimum threshold increase', parseFloat, 0)
+  .option(
+    '-t, --tolerance <tolerance>',
+    'keeps the new threshold below the current coverage',
```

Comment:
I'd prefer to describe the option _value_ instead:

```suggestion
    'threshold difference from actual coverage',
```

Please remember to update the README accordingly—I recommend running `jest-it-up --help` and copy-pasting the output there.